src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -1,14 +1,21 @@
 import React, { useContext } from 'react';
 import { arrayOf, shape, number } from 'prop-types';
 import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { pathOr } from 'ramda';
 
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import StoryPromo from '../StoryPromo';
 import CpsOnwardJourney from '../CpsOnwardJourney';
 
 const FeaturesAnalysis = ({ content, parentColumns }) => {
-  const { dir } = useContext(ServiceContext);
+  const { dir, translations } = useContext(ServiceContext);
+
+  const title = pathOr(
+    'Features & Analysis',
```

Comment:
Do we need the escaped & (`&amp`) in the fallback like we used to have below: `Features &amp; Analysis`?

Also we should probably add some new tests for the fallback of the `pathOr` in each container.

Reply:
(I don't know the answer to this but would be worth checking)

Reply:
In regards to the `&amp`, I've tested it locally and using `&` doesn't cause any issues.